terraform/access-management.tf

```
@@ -0,0 +1,19 @@
+locals {
+  am_non_prod_domains = ["mail-am-nonprod.platform.hmcts.net"]
+  am_prod_domains     = ["mail-am.platform.hmcts.net"]
+}
+
+module "access-management" {
```

applies throughout, https://www.terraform-best-practices.com/naming

> Use _ (underscore) instead of - (dash) everywhere (in resource names, data source names, variable names, outputs, etc).

```suggestion
module "access_management" {
```

---

done